COMMANDS.MD

```diff
@@ -109,3 +109,16 @@ indexer:anime
 Example: `indexer:manga`
 
 This simply translates to running the indexer without any additional configuration.
+
+#### Indexer: Incremental
+Incrementally indexes media entries from MAL.
+This command will compare the latest version of MAL ids from the [mal_id_cache](https://github.com/purarue/mal-id-cache)
+github repository and compares them with the downloaded ids from the previous run. If no ids found from the previous run, a full indexing session is started.
+
+Command:
+```
+indexer:incremental {mediaType*}
```

Comment:
Would writing it as `indexer:incremental {anime|manga}` make it more understandable for first timers? Since there's just 2 options.

Reply:
Nevermind, I missed that it's an array.